docs/manual/manual_safety_systems.rst

```diff
@@ -0,0 +1,264 @@
+
+Suppose that your :ref:`commanded drive voltage <manual_velocity_control_mechanisms>` is 2V, and the current derate is 0.4. 
+The actual drive voltage applied will be (2V * 0.4) = 0.8V. A basic illustration of derate application is shown below
+
+.. image:: ../_static/manual_images/safety/basic_derate_example.png
+    :align: center
+    :scale: 50%
+
+There are three separate derate values that are internally updated, each linked with a different measured module parameter (e.g. speed and temperature). There is a speed derate used to avoid mechanical failure, 
+a microcontroller temperature based derate to avoid microcontroller overheating, and a coil temperature derate to avoid coil overheating. 
+These three derate values are combined and applied to the target command voltage in order to protect your module. Details of each of these derate values as well as 
+how they are combined are discussed below.
+
+Speed Based Derate
+---------------------
+As mentioned above, your module's speed derate is meant to protect your module from mechanical failure. Speed derate is calculated using the module's current speed, 
+brushless drive's ``motor_redline_start`` parameter, and brushless drive's ``motor_redline_end`` parameter. You can find the redline parameters listed in the 
+:ref:`Brushless Drive Message Table <brushless_drive>`.
+
+You can also find these parameters in the Advanced tab as ``Speed Redline Start`` and ``Speed Redline End``.
+
+.. warning::
+    The redline values preloaded on your module have been tuned specifically to keep your module safe. Before adjusting these parameters, we highly recommend 
+    that you reach out to us at support@vertiq.co.
+
+.. image:: ../_static/manual_images/safety/speed_redline_control_center.png
+    :align: center
+    :scale: 50%
+
+
+The total speed derate is calculated by :math:`\text{Speed Derate} = \frac{\text{motor_redline_end - speed}}{\text{motor_redline_end - motor_redline_start}}` where the solution is saturated to [0,1].
+
+This means that for any speed below the redline start, there is no derate applied. For any speed at or above the redline end, 
+the speed derate is 0. Any speed in that range is subject to a variable derate [0, 1].
+
+Suppose your module's redline start is 1750 rad/s and its redline end is 2000 rad/s. This means that there will be no derate limiting applied under 1750 rad/s, 
+your module will stop spinning at and above 2000 rad/s, and will be derate limited in between.
+
+A graph illustrating the speed derate calculation is provided below:
+
+.. image:: ../_static/manual_images/safety/speed_derate_graph.png
+    :align: center
+    :scale: 50%
+
+Microcontroller Temperature Based Derate
+--------------------------------------------
+As mentioned above, your module's microcontroller temperature derate is meant to protect your module's microcontroller from overheating. 
```

Comment:
We should have a warning in this section that notes that this temperature derate only works when motoring.  Successful thermal derating is not guaranteed when generating.